services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -47,6 +47,18 @@
 logger = logging.getLogger(__name__)
 
 
+def _log_keep_message(message: str) -> None:
+    """
+    Logs a message which is meant to show in production deployments.
+    These messages are supposed to help debug the system.
+    """
+    logger.warning("[HINT] %s", message)
```

Comment:
I think changing the log level of the application is a better approach than hacking this inside the code.
And if that is for debugging only, I think it would even make sense to change the log level of that particular application during the debugging time. If that is because of the jupyter lab being killed issue, I think that is better to change temporarily the log level than add warnings that aren't warnings.